libs/top_level_metrics.py

```diff
@@ -121,9 +144,22 @@ def calculate_or_copy_test_positivity(data):
         cumulative_negative_tests = series_utils.interpolate_stalled_and_missing_values(
             data[CommonFields.NEGATIVE_TESTS]
         )
-        return calculate_test_positivity(cumulative_positive_tests, cumulative_negative_tests)
+        method = _lookup_test_positivity_method(
+            ts.provenance.get(CommonFields.POSITIVE_TESTS),
+            ts.provenance.get(CommonFields.NEGATIVE_TESTS),
+            log,
+        )
+        return (
+            calculate_test_positivity(cumulative_positive_tests, cumulative_negative_tests),
+            method,
+        )
     else:
-        return test_positivity
+        provenance = ts.provenance.get(CommonFields.TEST_POSITIVITY)
+        method = TestPositivityRatioMethod.get(provenance)
+        if method is None:
+            log.info("Unable to find TestPositivityRatioMethod", provenance=provenance)
```

Comment:
Info feels like a dangerous move here.  Do we ever expect this to happen?

Reply:
Do you mean you'd like this to be warning so we are less likely to overlook it breaking? SGTM.